### ISSUE TITLE
Enable logging

### ISSUE THREAD

**Issue by JustusAdam** [fsr/ifsr_course_enrollment#24]
There are a lot of print statements in the code. It'd be a good idea replacing those with a logger.

Extensive logger usage is okay and should be encouraged, but perhaps not using 'print'.

**Comment by fer-rum** [comment]
Agreed. It would be nice to see which tutor or organizer did what, to help recovering from when things get messed up.

**Comment by JustusAdam** [comment]
I might do that when I'm finished testing if my folder renaming worked
